cmd/relayer/start/start.go

```diff
@@ -54,9 +51,10 @@ func Start() *cobra.Command {
 
 			updateClientsCmd := relayer.GetUpdateClientsCmd()
 			utils.RunCommandEvery(updateClientsCmd.Path, updateClientsCmd.Args[1:], 5, logFileOption)
```

Comment:
no reason to run `updateClientsCmd` every 5 sec
1. it will fail if no state update happened meanwhile (and it happens every 100s)
2. there's no need for it at this frequency. we added this for the case of no transactions more than the IBC channel expiration time.

Reply:
have to disagree on that for the following reasons:
1. It also relays headers from hub to rollapp (so not every 100s but every new hub header).
2. we can't time the updateClient from rollapp to hub - i.e if a rollapp batch submission just got finalized, we want the headers to be accepted at the dymint light client on the hub ASAP. if you do it every 100s, you can fall into the gap where the headers of the rollapp are finialized in the hub, and you only send update of headers after 100s.
3. Don't see an obvious downside (performance wise/cpu wise) of running this action as it's mostly IO bound and most time is spent on querying and waiting for responses.